File: web_manager/src/main/webapp/js/service/seckillOrderService.js
//服务层
app.service('seckillOrderService',function($http){

    //读取列表数据绑定到表单中
    this.findAll=function(){
        return $http.get('../seckillOrderService/findAll.do');
    }

    this.search = function(searchEntity){
        return $http.post("../seckillOrderService/search.do?",searchEntity);
    }

});

File: pyg_interface/src/main/java/cn/itcast/core/service/SeckillOrderService.java
package cn.itcast.core.service;


import cn.itcast.core.pojo.seckill.SeckillOrder;


import java.util.List;

public interface SeckillOrderService {
    List<SeckillOrder> findAll();

    List<SeckillOrder> search(SeckillOrder seckillOrder);
}
